File: alphatimer-protocol-api/src/main/java/org/lisasp/alphatimer/api/protocol/events/messages/enums/KindOfTime.java
package org.lisasp.alphatimer.api.protocol.events.messages.enums;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
@Getter
public enum KindOfTime {
    Start('S'),
    SplitTime('I'),
    Finish('A'),
    TakeOverTime('D'),
    ReactionTimeAtStart('R'),
    Empty(' '),
    UnknownValueB('B');

    private final char value;
}

File: protocol-api/src/main/java/org/lisasp/alphatimer/api/protocol/events/messages/enums/KindOfTime.kt
package org.lisasp.alphatimer.api.protocol.events.messages.enums

enum class KindOfTime(val value: Char) {
    Start('S'),
    SplitTime('I'),
    Finish('A'),
    TakeOverTime('D'),
    ReactionTimeAtStart('R'),
    Empty(' '),
    UnknownValueB('B');
}
